.github/workflows/janitor.yml

```diff
@@ -0,0 +1,15 @@
+name: Janitor
+# Janitor cleans up previous runs of various workflows
+on:
+  pull_request:
+
+jobs:
+  cancel:
+    name: "Cancel Previous Runs"
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          workflow_id: 1041851,1401230,2837803
```

Comment:
I don't understand this number, is it important?

Reply:
these are taken from here https://api.github.com/repos/tendermint/tendermint/actions/workflows. Will add a comment with them